src/main/resources/words_en.properties

```diff
@@ -0,0 +1,45 @@
+GREETING_TEXT = I will calculate for free the cost of vehicle collection in Belarus for passenger vehicles (category M1).\n \
+  Choose where you are taking the car from:
+
+EAES_DETAILS = - for cars from Russia and EAEU;
+OTHER_COUNTRIES_DETAILS = - for cars from other countries.
+
+PHYSICAL_OR_JURIDICAL = Who enters the car: an individual or a legal entity?\n 
+PHYSICAL_PERSON_DETAILS = - if an individual entity;\n\ 
+JURIDICAL_PERSON_DETAILS = - if a legal entity.
+
+AGE_OF_AUTO = What is the age of the car?\n
+LESS_3_YEARS_DETAILS = - up to 3 years;\n
+BETWEEN_3_AND_7_YEARS_DETAILS = - from 3 to 7 years;\n
+MORE_7_YEARS_DETAILS = - over 7 years.
+
+GAS_OR_ELECTRIC_ENGINE = What type of engine?\n
```

Comment:
I'm not sure that it's good practice to store `\n` in localization files.
What do you think about moving it to java code?

Reply:
I think, in java is better. I just leave some \n in localization file because it seems good - in this case we don't need to split string for three variables 
![Clip2net_231018213320](https://github.com/Y-Pazniak/custom_calculator_chatbot/assets/73634869/9c547fe2-62e5-4c9d-a8e1-bc82124403b3)